internal/scheduleringester/dbops.go

```diff
@@ -119,15 +122,15 @@ type (
 	UpdateJobSetPriorities     map[JobSetKey]int64
 	MarkJobSetsCancelRequested map[JobSetKey]*JobSetCancelAction
 	MarkJobsCancelRequested    map[string]bool
-	MarkJobsCancelled          map[string]bool
+	MarkJobsCancelled          map[string]time.Time
 	MarkJobsSucceeded          map[string]bool
 	MarkJobsFailed             map[string]bool
 	UpdateJobPriorities        map[string]int64
 	UpdateJobSchedulingInfo    map[string]*JobSchedulingInfoUpdate
 	UpdateJobQueuedState       map[string]*JobQueuedStateUpdate
-	MarkRunsSucceeded          map[uuid.UUID]bool
+	MarkRunsSucceeded          map[uuid.UUID]time.Time
 	MarkRunsFailed             map[uuid.UUID]*JobRunFailed
-	MarkRunsRunning            map[uuid.UUID]bool
+	MarkRunsRunning            map[uuid.UUID]time.Time
```

Comment:
Not Sure what the `bool` was being used for? I didn't find it being used anywhere. This why I replaced the timestamp value so we can consume it in the schedulerdb

Reply:
I originally wanted to use a set, but there isn't a set datatype in go so I used map[string]bool. Using time instead makes sense for this change.